qiskit/visualization/qcstyle.py

```diff
@@ -96,8 +96,8 @@ def __init__(self):
             'meas': non_gate_color
         }
         self.latexmode = False
-        self.fold = None  # To be removed after 0.10 is released
```

Comment:
according to this comment, `fold` should just be removed from `qcstyle`. it is now a top-level arg to `circuit.draw()`.

Reply:
Ok. Here are 3 outputs. Do you want me to remove all references to fold in qcstyle?

![image](https://user-images.githubusercontent.com/16268251/79283699-748aa600-7e6d-11ea-8ac1-1aadaaa3026f.png)
![image](https://user-images.githubusercontent.com/16268251/79283783-b9aed800-7e6d-11ea-8722-b06616e0e837.png)
![image](https://user-images.githubusercontent.com/16268251/79283810-d0552f00-7e6d-11ea-86fc-e29d3913f0ee.png)